content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -12,11 +12,11 @@ title: 'Assessment: Classes and objects'
 
 ## Note about submission format
 
-On Tilde you'll notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
+On Tilde you will notice that this card is asking for a link submission. **Please don't worry about submitting a link**. You will be assessed according to {{% contentlink path="specific-skill-success-criteria/introduction-to-assessments" %}}
 
 
 
-All students need to provably understand all of the following concepts:
+All students need to understand and prove the following concepts:
```

Comment:
"You need to prove that you understand the following concepts" would be better. 

I'm not sure that  a person can prove "composition"